### ISSUE TITLE
Copying `fastlane env` to clipboard fails with [!] undefined method `create' for Tempfile:Class (NoMethodError)

### ISSUE THREAD

**Issue by 0xced** [fastlane/fastlane#6945]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

I launched `fastlane env` and in the end I tried to copy the results to the clipboard but it failed.

##### Complete output when running fastlane, including the stack trace and command used

$ fastlane env
[14:13:58]: Generating fastlane environment output, this might take a few seconds...
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                 | Value                                                          |
| ------------------- | -------------------------------------------------------------- |
| OS                  | 10.11.6                                                        |
| Ruby                | 2.0.0                                                          |
| Bundler?            | false                                                          |
| Xcode Path          | /Applications/Xcode81.app/Contents/Developer/                  |
| Xcode Version       | 8.1                                                            |
| Git                 | git version 2.9.3                                              |
| Installation Source | /Users/0xced/.gem/bin/fastlane                                 |
| Host                | Mac OS X 10.11.6 (15G1108)                                     |
| Ruby Lib Dir        | /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib |
| OpenSSL Version     | OpenSSL 0.9.8zc 19 Mar 2015                                    |

### fastlane files:

**No Fastfile found**

**No Appfile found**

### fastlane gems

| Gem                 | Version | Update-Status |
| ------------------- | ------- | ------------- |
| credentials_manager | 0.16.2  | ✅ Up-To-Date  |
| spaceship           | 0.37.0  | ✅ Up-To-Date  |
| fastlane_core       | 0.55.0  | ✅ Up-To-Date  |
| deliver             | 1.15.0  | ✅ Up-To-Date  |
| snapshot            | 1.16.3  | ✅ Up-To-Date  |
| frameit             | 3.0.0   | ✅ Up-To-Date  |
| pem                 | 1.4.0   | ✅ Up-To-Date  |
| cert                | 1.4.3   | ✅ Up-To-Date  |
| sigh                | 1.11.2  | ✅ Up-To-Date  |
| produce             | 1.3.0   | ✅ Up-To-Date  |
| gym                 | 1.12.0  | ✅ Up-To-Date  |
| pilot               | 1.12.0  | ✅ Up-To-Date  |
| scan                | 0.14.1  | ✅ Up-To-Date  |
| supply              | 0.7.1   | ✅ Up-To-Date  |
| match               | 0.10.0  | ✅ Up-To-Date  |
| screengrab          | 0.5.5   | ✅ Up-To-Date  |
| fastlane            | 1.107.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| highline                  | 1.7.8        |
| commander                 | 4.4.0        |
| terminal-table            | 1.4.5        |
| json                      | 1.8.3        |
| multipart-post            | 2.0.0        |
| faraday                   | 0.9.2        |
| faraday_middleware        | 0.10.0       |
| dotenv                    | 2.1.1        |
| net-ssh                   | 3.2.0        |
| net-sftp                  | 2.1.2        |
| plist                     | 3.1.0        |
| rubyzip                   | 1.1.7        |
| security                  | 0.1.3        |
| krausefx-shenzhen         | 0.14.10      |
| slack-notifier            | 1.5.1        |
| CFPropertyList            | 2.3.3        |
| claide                    | 1.0.1        |
| colored                   | 1.2          |
| nanaimo                   | 0.2.2        |
| xcodeproj                 | 1.4.1        |
| rouge                     | 1.11.1       |
| xcpretty                  | 0.2.4        |
| terminal-notifier         | 1.7.1        |
| public_suffix             | 2.0.4        |
| addressable               | 2.5.0        |
| credentials_manager       | 0.16.2       |
| multi_xml                 | 0.5.5        |
| unf_ext                   | 0.0.7.2      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20161021 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| xcode-install             | 2.0.8        |
| word_wrap                 | 1.0.0        |
| multi_json                | 1.12.1       |
| babosa                    | 1.0.2        |
| excon                     | 0.54.0       |
| gh_inspector              | 1.0.2        |
| fastlane_core             | 0.55.0       |
| bundler                   | 1.13.6       |
| fastimage                 | 1.6.8        |
| mini_magick               | 4.5.1        |
| xcpretty-travis-formatter | 0.0.4        |
| uber                      | 0.0.15       |
| representable             | 2.3.0        |
| retriable                 | 2.1.0        |
| hurley                    | 0.2          |
| little-plugger            | 1.1.4        |
| logging                   | 2.1.0        |
| jwt                       | 1.5.6        |
| memoist                   | 0.15.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.2.4      |
| google-api-client         | 0.9.20       |
| growl                     | 1.0.3        |
</details>

*generated on:* **2016-11-10**
</details>

```
🙄  Wow, that's a lot of markdown text... should fastlane put it into your clipboard, so you can easily paste it on GitHub? (y/n)
y

Looking for related GitHub issues on fastlane/fastlane...

Found no similar issues. To create a new issue, please visit:
https://github.com/fastlane/fastlane/issues/new
Run `fastlane env` to append the fastlane environment to your issue
/Users/0xced/.gem/gems/fastlane-1.107.0/lib/fastlane/environment_printer.rb:205:in `copy_to_clipboard': [!] undefined method `create' for Tempfile:Class (NoMethodError)
	from /Users/0xced/.gem/gems/fastlane-1.107.0/lib/fastlane/environment_printer.rb:7:in `output'
	from /Users/0xced/.gem/gems/fastlane-1.107.0/lib/fastlane/commands_generator.rb:252:in `block (2 levels) in run'
	from /Users/0xced/.gem/gems/commander-4.4.0/lib/commander/command.rb:178:in `call'
	from /Users/0xced/.gem/gems/commander-4.4.0/lib/commander/command.rb:178:in `call'
	from /Users/0xced/.gem/gems/commander-4.4.0/lib/commander/command.rb:153:in `run'
	from /Users/0xced/.gem/gems/commander-4.4.0/lib/commander/runner.rb:444:in `run_active_command'
	from /Users/0xced/.gem/gems/fastlane_core-0.55.0/lib/fastlane_core/ui/fastlane_runner.rb:38:in `run!'
	from /Users/0xced/.gem/gems/commander-4.4.0/lib/commander/delegates.rb:15:in `run!'
	from /Users/0xced/.gem/gems/fastlane-1.107.0/lib/fastlane/commands_generator.rb:257:in `run'
	from /Users/0xced/.gem/gems/fastlane-1.107.0/lib/fastlane/commands_generator.rb:31:in `start'
	from /Users/0xced/.gem/gems/fastlane-1.107.0/lib/fastlane/cli_tools_distributor.rb:58:in `take_off'
	from /Users/0xced/.gem/gems/fastlane-1.107.0/bin/fastlane:5:in `<top (required)>'
	from /Users/0xced/.gem/bin/fastlane:23:in `load'
	from /Users/0xced/.gem/bin/fastlane:23:in `<main>'
```

**Comment by hjanuschka** [comment]
seems to be some weird bug because of ruby 2.0 - see http://stackoverflow.com/questions/35919011/undefined-method-create-for-tempfileclass

**Comment by 0xced** [comment]
@hjanuschka Actually, Tempfile::create is simply [not available on ruby 2.0.0](http://ruby-doc.org/stdlib-2.0.0/libdoc/tempfile/rdoc/Tempfile.html).
